packages/client/src/Call.ts

```diff
@@ -1392,6 +1399,19 @@ export class Call {
     });
   };
 
+  /**
+   * Query call members with filter query. The result won't be stored in call state.
+   * @param request
+   * @returns
+   */
+  queryMembers = (request: Omit<QueryMembersRequest, 'type' | 'id'>) => {
```

Comment:
I wonder why `type` and `id` aren't part of the endpoint path...

Reply:
I think because that endpoint is already used for call member updates: https://getstream.github.io/protocol/?urls.primaryName=Video#/Video%20Calls/UpdateCallMembers - but maybe that could be a `PATCH` endpoint

Reply:
yeah, it could be... but on our side, I think we are covered for any potential future changes because we exclude `id` and `type` from the `request`.